core/src/helpers/mod.rs

```diff
@@ -244,8 +251,8 @@ mod tests {
     #[test]
     fn test_single_word() {
         assert_eq!(to_pascal_case("user"), "User");
-        assert_eq!(to_pascal_case("CONSTANT"), "Constant");
-        assert_eq!(to_pascal_case("URI"), "Uri");
+        assert_eq!(to_pascal_case("CONSTANT"), "CONSTANT");
+        assert_eq!(to_pascal_case("URI"), "URI");
```

Comment:
the idea with this was to make it Constant or Uri any reason test here changed?

Reply:
Yeah, basically something in the `sol!` macro changes and it changed the casing for full uppercase single words to remain as such, so this util needs to match that logic since it's used to generate code which must match the  contract `sol!` output.